app/src/main/java/ch/hikemate/app/model/route/HikeRoute.kt

```diff
@@ -130,6 +131,21 @@ data class HikeRoute(
     val ways: List<LatLong>,
     val name: String? = null,
     val description: String? = null
-)
+) {
+
+  private val colors =
+      listOf(
+          0xFF7E57C2.toInt(), // Purple
+          0xFF1E88E5.toInt(), // Blue
+          0xFFFFA726.toInt(), // Orange
+          0xFFEF5350.toInt(), // Red
+          0xFF26A69A.toInt() // Teal
+          )
```

Comment:
Suggestion: could those colors be moved to the `Color.kt` file of the project? Or maybe we have a task for this, I'm not sure.

Reply:
I will do it now